### ISSUE TITLE
Error when handling Svelte code

### ISSUE THREAD

**Issue by markjaquith** [shikijs/shiki#196]
This rather standard Svelte code causes an error:

```js
const shiki = require('shiki')

shiki.getHighlighter({
    theme: 'nord'
})
.then(highlighter => {
    console.log(highlighter.codeToHtml(`<button on:click={increment}>Click</button>`, 'svelte'))
})
```

> `UnhandledPromiseRejectionWarning: Error: 105,110,118,97,108,105,100,32,112,97,116,116,101,114,110,32,105,110,32,108,111,111,107,45,98,101,104,105,110,100
    at OnigScanner.findNextMatchSync (/Users/mark/git/shiki-svelte-bug/node_modules/onigasm/lib/OnigScanner.js:100:27)
    at /Users/mark/git/shiki-svelte-bug/node_modules/vscode-textmate/release/main.js:1:9891
    at R (/Users/mark/git/shiki-svelte-bug/node_modules/vscode-textmate/release/main.js:1:10242)
    at f (/Users/mark/git/shiki-svelte-bug/node_modules/vscode-textmate/release/main.js:1:12144)
    at S (/Users/mark/git/shiki-svelte-bug/node_modules/vscode-textmate/release/main.js:1:11997)
    at w (/Users/mark/git/shiki-svelte-bug/node_modules/vscode-textmate/release/main.js:1:9336)
    at f (/Users/mark/git/shiki-svelte-bug/node_modules/vscode-textmate/release/main.js:1:13085)
    at S (/Users/mark/git/shiki-svelte-bug/node_modules/vscode-textmate/release/main.js:1:11997)
    at e._tokenize (/Users/mark/git/shiki-svelte-bug/node_modules/vscode-textmate/release/main.js:1:8354)
    at e.tokenizeLine2 (/Users/mark/git/shiki-svelte-bug/node_modules/vscode-textmate/release/main.js:1:7575)`

**Comment by PuruVJ** [comment]
Facing exact same issue 👍

**Comment by Valexr** [comment]
in this repo https://github.com/shikijs/shiki-playground shiki uses in frontend part...
set WASM & CND – https://github.com/shikijs/shiki-playground/blob/main/src/highlighter.ts
& initialize app after shiki  – https://github.com/shikijs/shiki-playground/blob/main/src/main.ts

**Comment by bluwy** [comment]
This seems to be fixed in the latest version (0.10.0), not getting the errors anymore.

**Comment by markjaquith** [comment]
This issue is not happening anymore in `0.9.15` and as @bluewy above reports it's also fixed in `0.10.0`, I'm closing this as resolved!
